addons/l10n_mx/data/l10n_mx_chart_data.xml

```diff
@@ -144,22 +144,30 @@
         <field name="reconcile" eval="True"/>
         <field name="tag_ids" eval="[(6,0,[ref('account_tag_205_01')])]"/>
     </record>
-    <record id='cuenta206_01' model='account.account.template'>
+    <record id='cuenta206_01_01' model='account.account.template'>
         <field name='name'>Anticipo de cliente nacional</field>
         <field name='code'>206.01.01</field>
         <field name="chart_template_id" ref="mx_coa"/>
-        <field name="user_type_id" ref="account.data_account_type_payable"/>
+        <field name="user_type_id" ref="account.data_account_type_current_liabilities"/>
         <field name="reconcile" eval="True"/>
         <field name="tag_ids" eval="[(6,0,[ref('account_tag_206_01')])]"/>
     </record>
-    <record id='cuenta206_02' model='account.account.template'>
+    <record id='cuenta206_02_01' model='account.account.template'>
```

Comment:
Hello @GavyMG 

Could you avoid change the xml_id in the stable version.

What do you think @hbto ?

Regards

Reply:
I agree @luistorresm xml_id must be kept.

regards